src/internal/connector/sharepoint/pageInfo.go

```diff
@@ -0,0 +1,44 @@
+package sharepoint
+
+import (
+	"time"
+
+	bmodel "github.com/microsoftgraph/msgraph-beta-sdk-go/models"
+
+	"github.com/alcionai/corso/src/pkg/backup/details"
+)
+
+// sharePointPageInfo propagates metadata from  the SharePoint Page data type
+// into searchable content.
+// Page Details: https://learn.microsoft.com/en-us/graph/api/resources/sitepage?view=graph-rest-beta
+func sharePointPageInfo(page bmodel.SitePageable, size int64) *details.SharePointInfo {
```

Comment:
Issue #2071 is to refactor `SharePoint` into 3 distinct items similar to exchange.
- Pages
- Lists
- Libraries 
This will constitute a site being effectively backed up. Not USED here; however, it was the quickest way to verify library complications for using a single library from `msgraph-beta-sdk-go`

Reply:
For completion of issue #2072, the `SharePoint.Page` will be placed outside of the onedrive.